source/ways-of-working/new-component/shutter.html.md.erb

```diff
@@ -0,0 +1,138 @@
+---
+title: Shutter Implementation and Design
+last_reviewed_on: 2020-02-07
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+## Purpose
+The purpose of this document is to describe the shutter solution design and its end to end implementation.
+
+## Solution
+
+![Shutter Solution Design Docs](/images/Shuttersolution_v2.png)
```

Comment:
does this need v2 in it?

```suggestion
![Shutter Solution Design Docs](/images/shutter-solution.png)
```

Reply:
I included v2 as we already had a solution earlier with shutter and this is a new solution

Reply:
but it's going away and this is only for the new solution, and I don't think you've documented the old one?

Reply:
I am not sure if the earlier version of the solution was documented but the program already knows about the 1st solution that is there and this will be a completely new solution that makes it a version 2.